build-tools-internal/src/main/groovy/elasticsearch.ide.gradle

```diff
@@ -131,7 +131,7 @@ if (providers.systemProperty('idea.active').getOrNull() == 'true') {
       ':server:generateModulesList',
       ':server:generatePluginsList',
       ':generateProviderImpls',
-      ':libs:elasticsearch-native:elasticsearch-native-libraries:extractLibs',
+      ':libs:native:libraries:extractLibs',
```

Comment:
Why has `elasticsearch-native-libraries` becomes simply `libraries`?

Reply:
It's just the local project name. Before we manually changed it to `elasticsearch-native-libraries` in `settings.gradle`. Now the artifact remains the same (set recursively in libs/build.gradle), but the local name is simplified.